lib/codetogether/games.ex

```diff
@@ -37,10 +39,19 @@ defmodule Codetogether.Games do
     Repo.get_by(Nominee, %{body: name, game_id: game.id})
   end
 
+  @spec create_entry(Codetogether.Games.Game.t(), String.t()) :: any
   def create_entry(%Game{} = game, addition) do
-    %Entry{}
-    |> Entry.changeset(%{game_id: game.id, body: addition})
-    |> Repo.insert()
+    case addition do
+      ":bk" ->
+        Entry
+        |> Ecto.Query.last
+        |> Repo.one
+        |> Repo.delete
```

Comment:
Would this delete an `Entry` on another Game if it was running at the same time?

Reply:
Wonder if we should just add a `deleted: true` flag to a deleted entry and not pull those from the db when we make the code.